integration/meta-test.ts

```diff
@@ -441,9 +441,9 @@ test.describe("v2_meta", () => {
         `,
 
         "app/routes/index.jsx": js`
-          export const meta = ({ data, matches }) => [
-            ...matches.map((match) => match.meta),
-          ];
+          export const meta = ({ data, matches }) =>
+            matches.flatMap((match) => match.meta);
```

Comment:
While both technically work (React renders nested arrays of arrays just fine), the output of `map` doesn't match our return type API.